### PR TITLE
Admin send mail statistic

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/75160

## WHAT (optional)
- Add email sending statistics feature at the end of each month

## HOW
- Add email sending statistics feature at the end of each month

## Evidence (Screenshot or Video)
Screenshot from 2024-05-15 10-26-23
Screenshot from 2024-05-15 10-26-10

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/services/user.service.ts [hunk 1]
```diff
@@ -75,19 +75,31 @@ export const getUsers = async (data: any) => {
...
+  //   });
+  // }
+
+  // if (data.status === EntityStatus.ACTIVE) {
+  //   query.andWhere('user.isActive = true');
+  // }
+
+  // if (data.status === EntityStatus.INACTIVE) {
+  //   query.andWhere('user.isActive = false');
+  // }
```
Comment (huongnt-2545) [inline src/services/user.service.ts]: sao code chỗ này lại comment e nhỉ?

Comment (ghost) [reply src/services/user.service.ts]: em bỏ đoạn này quên chưa xóa ạ

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**